.infrastructure/env/prod/terraform.tfvars

```diff
@@ -1,3 +1,4 @@
 aws_region        = "eu-south-1"
 environment       = "prod"
 github_repository = "pagopa/b2b-portals"
+# dns_domain_name   = "b2b.pagopa.it" ## Uncommented and Change this value when available dns domain
```

Comment:
```suggestion
```
I'd remove this just because we don't know yet what will be the domain, and I think it could be misleading if someone see this domain